MaterialComponentsAlpha.podspec

```diff
@@ -47,6 +47,14 @@ Pod::Spec.new do |mdc|
     extension.dependency "MaterialComponents/schemes/Typography"
   end
 
+  # Banners
+
+  mdc.subspec "Banners" do |component|
+    component.ios.deployment_target = '8.0'
+    component.public_header_files = "components/#{component.base_name}/src/*.h"
+    component.source_files = "components/#{component.base_name}/src/*.{h,m}", "components/#{component.base_name}/src/private/*.{h,m}"
+  end
```

Comment:
We're working on the guidance here still, but components won't become Alpha components until they've passed API review. Please remove this subspec for now.

Reply:
I see. Thanks for pointing it out! Addressed in 13e12dd